src/commands/retrieve.ts

```diff
@@ -247,6 +247,7 @@ export default function retrieve(context: vscode.ExtensionContext, resource?: vs
                         if (option && option.description) {
                             name = name.replace(option.description + path.sep, '');
                         }
+                        name = path.normalize(name);
```

Comment:
You need to move this to line 248 before you replace otherwise the replace will not work which creates the issue